.github/workflows/continuous-integration-windows.yml

```diff
@@ -31,4 +31,26 @@ jobs:
     - name: build library
       shell: bash
       run: |
-        cmake --build build --parallel 2 --config Release
+        cmake --build build --parallel 4 --config Release
+  windows-serial:
+    # Serial build on Windows
+    name: Windows Serial
+    runs-on: windows-2019
+
+    steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+    - name: configure
+      shell: bash
+      run: |
+        mkdir build
+        mkdir c:/project
+        cd build
+        cmake -DKokkos_ENABLE_Serial=ON -DKokkos_ENABLE_TESTS=ON ..
+    - name: build library
+      shell: bash
+      run: |
+        cmake --build build --parallel 4 --config Release
```

Comment:
So Visual Studio is a multi configuration generator?

Reply:
Yeah, unfortunately lol